debezium-core/src/main/java/io/debezium/relational/RelationalDatabaseConnectorConfig.java

```diff
@@ -234,7 +234,7 @@ public static DecimalHandlingMode parse(String value, String defaultValue) {
             .withType(Type.LIST)
             .withWidth(Width.LONG)
             .withImportance(Importance.LOW)
-            .withValidation(Field::isListOfRegex, RelationalDatabaseConnectorConfig::validateTableExcludeList)
+            .withValidation(Field::isListOfRegex, RelationalDatabaseConnectorConfig::validateTableBlacklist)
```

Comment:
Not "Blacklist" it should always be Exclude/Include. Other occurrences as well!

Reply:
A separate validation step is needed for old and new option names; hence the separate method and according name.